cmd/user-ssh-keys-agent/README.md

```diff
@@ -1 +1,18 @@
+#User SSH Keys Agent
+This document explains the functionality of the user ssh keys agent, that is deployed in the kubermatic user clusters.
+
+###Functionality
+The user ssh key agent is responsible for syncing the defined user ssh keys to the worker nodes, when users
+attach ssh keys to the user clusters. When users choose to add a user ssh key to a cluster after it was created 
+the agent will sync those keys to the worker machines by fetching the ssh keys and write them to the `authorized_keys` 
+file on each worker node. The agent runs as a daemonset in the cluster. If a user changes the `authorized_keys` file 
+manually via logging into machine and change the content of the file the agent will reject the changes and will rewrite 
+the content of the file based on the attached user ssh keys.
+
+The agent is deployed to the user clusters by default and it is not possible to change whether to deploy it or not once the cluster 
+has been created. If users decide to disable the agent, they should take care of adding ssh keys to the worker nodes by themselves.
```

Comment:
The most interesting question that will come up in half a year is "_WHY_ can this not be changed after the creation?" -- can you describe the reasons or point the reader to where they can find out more?